telemetry/definitions/commonDefinitions.json

```diff
@@ -402,6 +418,11 @@
             "unit": "Milliseconds",
             "metadata": [{ "type": "result" }]
         },
+        {
+            "name": "cloudformation_deploy",
+            "description": "Called when deploying a CloudFormation template",
+            "metadata": [{ "type": "result" }, { "type": "regionId" }, { "type": "initialDeploy" }]
```

Comment:
when did `initalDeploy` get into these metrics? What question does this answer?

Reply:
`initialDeploy` was introduced with beanstalk deploy, and signals whether a deploy is targeting a new destination (like a create), or an existing destination (like an update).

Reply:
I get that, but why do we care? It's the same API call and UI regardless?

Reply:
This provides insight into whether we are seeing more successes/failures around creates vs updates, as well as an understanding around whether Toolkits are used more for creates or updates.

Reply:
> It's the same API call and UI

These are dangerous assumptions to make across the Toolkits. In VS, CloudFormation template deployment uses the same entrypoints to deploy a new or updated stack, and the deploy wizard has some variations based on user inputs. 

The API calls used are not as relevant to the metric around the act of deployment.  [CreateStack](https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_CreateStack.html) or [UpdateStack](https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_UpdateStack.html) are used as needed.